.github/workflows/auto-cherry-picker.yaml

```diff
@@ -0,0 +1,33 @@
+name: Auto Cherry-Picker
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
```

Comment:
No, `workflow_dispatch` is only there because meh why not.

This triggers on all merged PRs right after they merge. It's the _auto_ in _auto-cherry-picker__

Reply:
Ah! Neat.

I think that the original author needs to be included as a reviewer in that case then.

Reply:
Ah good point

Reply:
Added